common/persistence/nosql/nosqlplugin/cassandra/gocql/consistency.go

```diff
@@ -80,3 +81,103 @@ func mustConvertSerialConsistency(c SerialConsistency) gocql.SerialConsistency {
 		panic(fmt.Sprintf("Unknown gocql SerialConsistency level: %v", c))
 	}
 }
+
+func (c Consistency) MarshalText() (text []byte, err error) {
```

Comment:
please add unit tests for the new functions here. would be great to cover the whole file

Reply:
unit tests are added now